src/pages/EditMonitor.vue

```diff
@@ -1000,9 +1010,20 @@
             <RemoteBrowserDialog ref="remoteBrowserDialog" />
         </div>
     </transition>
+    <div ref="modal" class="modal fade" tabindex="-1">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <textarea id="curlDebug" v-model="curlCommand" class="form-control mb-3" placeholder="curl -v 'https://example.com/healthcheck'" readonly></textarea>
+                    <button id="debug-copy-btn" class="btn btn-primary" type="button" @click.stop="copyToClipboard">{{ $t("Copy") }}</button>
```

Comment:
I don't quite like this button placement (it somehow seems a bit off, but I can't quite put my finger on it..). Experimenting with different ways of doing this have not led me to an immediate better solution.

<details><summary>Maybe a copy button in the top right might be bettter. What do you think? (click to expand)</summary>
<p>

![image](https://github.com/user-attachments/assets/6bcc2f40-df5e-473c-b4fb-62a05c3829b4)

</p>
</details>

Reply:
I like that idea, implemented.